pytest_marker_bugzilla.py

```diff
@@ -146,7 +146,7 @@ def pytest_runtest_setup(self, item):
         bugs = item.funcargs["bugs"]
         skippers = []
         for bz in bugs.bugs_gen:
-            if bz.status in ["NEW", "ASSIGNED", "ON_DEV"]:
+            if bz.status in ["NEW", "ASSIGNED", "ON_DEV", "MODIFIED", "POST"]:
```

Comment:
would be nice if this could be configurable via conftest.py or setup.cfg instead of hard-coding it like this. Other projects may want to tweak this and it is unfeasible to accommodate everything in this way

Reply:
Agreed, there may be users of the bugzilla plugin which expects this behavior.

Reply:
I agree, I think developer can take this as an RFE :)